reference/system-databases/cluster-load.md

```diff
@@ -0,0 +1,65 @@
+---
+title: CLUSTER_LOAD
+summary: Learn the `CLUSTER_LOAD` cluster load table.
+category: reference
+---
+
+# CLUSTER_LOAD
+
+The `CLUSTER_LOAD` cluster load table provides the current load information of the server where each node of the TiDB cluster is located.
```

Comment:
@reafans Please help confirm whether it's `instance` or `node` here.
```suggestion
The `CLUSTER_LOAD` cluster load table provides the current load information of the server where each instance of the TiDB cluster is located.
```

Reply:
It is instance here.